### ISSUE TITLE
Typo in template for BugReport-Issue

### ISSUE THREAD

**Issue by MonsterDruide1** [tudo-aqua/bgw#321]
Looking at the template for bug reports on this repo, the following line has a useless closing brace:
`Please complete the following information):`

Context:
```
[...]
**Screenshots**
If applicable, add screenshots to help explain your problem.

Please complete the following information):
 - OS: [e.g. Windows 10, Mac OS X, Ubuntu, etc.]
 - Java version [e.g. openJDK 11]
[...]

**Comment by MonsterDruide1** [comment]
Additionally, a colon should be added after
`- Java version`

along with a field to provide a stack trace, if applicable.

**Comment by dominikmaeckel** [comment]
Fixed in 6982c2409b45dfc4b4cda96866e3c53b3d2f17c3.

**Comment by MonsterDruide1** [comment]
Not fully completed: A field for StackTrace would still be missing, as I would think that will still be useful in a lot of bug reports.
